_wasm_crypto/mod.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+export * as crypto from "./crypto.js";
+
+/**
+ * All cryptographic hash/digest algorithms supported by std/_wasm_crypto.
+ *
+ * For algorithms that are supported by WebCrypto, the name here must match the
+ * one used by WebCrypto. Otherwise we should prefer the formatting used in the
+ * official specification. All names are uppercase to faciliate case-insensitive
+ * comparisons required by the WebCrypto spec.
+ */
+export const digestAlgorithms = [
+  "BLAKE2B-256",
+  "BLAKE2B-384",
+  "BLAKE2B",
+  "BLAKE2S",
+  "BLAKE3",
+  "KECCAK-224",
+  "KECCAK-256",
+  "KECCAK-384",
+  "KECCAK-512",
+  "SHA-384",
```

Comment:
wikipedia SHA-2 page seems putting SHA-384 in the same insecure category as SHA-{224,256,512} for length extendable attacks ref: https://en.wikipedia.org/wiki/SHA-2 Do you think SHA-384 is secure?

Reply:
That depends on your definition. Wikipedia's table points out that SHA-384 only provides 128 bits of collision resistance, which is less than the 384 bits that we should want from its digest size. However, in practice this is still only a theoretical concern, as 128 bits is still considered "secure" by most definitions, even if it's not efficient.

I asked a couple of friends who are cryptography professionals, and they both said they considered SHA-384 a safe recommendation, even in 2021, with the benefit of also being extremely widely-supported (such as by native WebCrypto), and the cost of efficiency.

(But if this is a point of disagreement I'm happy to just delete the comments since they're not really important.)

Reply:
Thank you for the input! Ok. Let's keep this as is.